docs/user/rstudio/ide/guide/pkg-devel/writing-packages.qmd

```diff
@@ -88,14 +94,8 @@ The Build and Reload command performs several steps in sequence to ensure a clea
 
 4.  Reloads the package in the new R session by executing the `library` function.
 
-Note that you can also execute **Build and Reload** using a keyboard shortcut <kbd>Ctrl+Shift+B</kbd> as well as configure RStudio to automatically save open source files prior to rebuiling.
-
-Note that the **Command Palette,** accessed by <kbd>Ctrl+Shift+P</kbd>, can also execute `devtools`, `usethis`, or Package Development specific commands:
-
-![](images/command-palette-pkgs.png)
+You can also execute **Build and Reload** using a keyboard shortcut <kbd>Ctrl+Shift+B</kbd> as well as configure RStudio to automatically save open source files prior to rebuiling.
```

Comment:
`Clean and Install`

Also note -- this would be Cmd + Shift + B on Macs
And I believe we've modified the default for the Keyboard shortcut -- Ctrl + Shift + B now just does Install (without Clean). I don't think there's an explicit keyboard shortcut for Clean and Install unless the user specifically sets one. 

You can ask @romainfrancois if you have questions about these changes